pontoon/base/views.py

```diff
@@ -158,9 +159,51 @@ def projects(request, template='projects.html'):
     return render(request, template, data)
 
 
-def translate(request, locale, slug, part=None, template='translate.html'):
+def locale_project(request, locale, slug):
+    """Locale-project overview."""
+
+    # Redirect to translate view
+    if request.GET.get('q'):
```

Comment:
Am I correct in thinking that this is to handle the search submission? If so, just make the form for the search submit directly to the translate view, this shouldn't be necessary.